File: Backend/grocapitalapp-django/core/urls.py
from django.urls import path, include
from .api import (
    RegisterAPI, 
    LinkedinRegisterAPI, 
    LoginAPI, 
    LinkedinLoginAPI, 
    ResetPasswordAPI, 
    CodeVerificationAPI, 
    ChangePasswordAPI, 
    TransactionAPI, 
    UserAPI, 
    UserProfileAPI, 
    EditUserProfileAPI, 
    BusinessInformationAPI, 
    PersonalInformationAPI, 
    FinancialInformationAPI, 
    KpiMetricsAPI, 
    GetAllData, 
    LoanHistoryAPI, 
    TransactionAPI, 
    UserPasswordResetAPI,
    PlansViewSet,
    UserPlanViewSet,
    UserPlanHistoryViewSet,
    UserDocumentationViewSet
    )
from knox import views as knox_views
from rest_framework import routers

router = routers.DefaultRouter()
# router.register('password-update', UserPasswordUpdateAPI, 'password-update')
router.register('user-profile', UserProfileAPI, 'user-profile')
router.register('edit-user-profile', EditUserProfileAPI, 'edit-user-profile')
router.register('business-information',
                BusinessInformationAPI, 'business-information')
router.register('personal-information',
                PersonalInformationAPI, 'personal-information')
router.register('financial-information',
                FinancialInformationAPI, 'financial-information')
router.register('kpi-metrics', KpiMetricsAPI, 'kpi-metrics')
router.register('get-all-data', GetAllData, 'get-all-data')
router.register('loan-history', LoanHistoryAPI, 'loan-history')
router.register('transaction', TransactionAPI, 'transaction')
router.register('plan', PlansViewSet, 'plan')
router.register('user-plan', UserPlanViewSet, 'user-plan')
router.register('user-plan-history', UserPlanHistoryViewSet, 'user-plan-history')
router.register('user-docs', UserDocumentationViewSet, 'user-docs')

urlpatterns = [
    path('api/auth', include('knox.urls')),
    path('api/auth/register', RegisterAPI.as_view()),
    path('api/auth/linkedin-register', LinkedinRegisterAPI.as_view()),
    path('api/auth/linkedin-login', LinkedinLoginAPI.as_view()),
    path('api/auth/login', LoginAPI.as_view()),
    path('api/auth/user', UserAPI.as_view()),
    path('api/auth/reset-password', ResetPasswordAPI.as_view()),
    path('api/auth/settings/reset-password', UserPasswordResetAPI.as_view()),
    path('api/auth/code-verification', CodeVerificationAPI.as_view()),
    path('api/auth/change-password', ChangePasswordAPI.as_view()),
    path('api/auth/logout', knox_views.LogoutView.as_view(), name='knox_logout'),
    path('api/', include(router.urls))
]

File: Frontend/grocapitalapp-react/src/services/util/urls.js
// DJANGO ENDPOINTS

// AUTH ENPOINTS
export const REGISTER_URL = 'auth/register'
export const LINKEDIN_REGISTER_URL = 'auth/linkedin-register'
export const LOGIN_URL = 'auth/login'
export const LINKEDIN_LOGIN_URL = 'auth/linkedin-login'
export const LOGOUT_URL = 'auth/logout'
export const AUTHORIZE_USER_TOKEN_URL = 'auth/user'
export const RESET_PASSWORD_URL = 'auth/reset-password'
export const CODE_VERIFICATION_URL = 'auth/code-verification'
export const PASSWORD_CHANGE_URL = 'auth/change-password'

// FORMS 
export const USER_PROFILE_URL = 'user-profile/'
export const EDIT_USER_PROFILE_URL = 'edit-user-profile/'
export const ADD_BUSINESS_INFORMATION_URL = 'business-information/'
export const ADD_PERSONAL_INFORMATION_URL = 'personal-information/'
export const ADD_FINANCIAL_INFORMATION_URL = 'financial-information/'
export const ADD_KPI_METRICS_URL = 'kpi-metrics/'
export const GET_ALL_DATA_USER_URL = 'get-all-data/'
export const USER_DOCUMENTATION = 'user-docs/'

//PLANS ENDPOINTS
export const USER_PLAN_URL = 'user-plan/'

// LOAN HISTORY AND TRANSACTIONS
export const GET_LOAN_HISTORY_DATA_URL = 'loan-history/'
export const GET_TRANSACTIONS_DATA_URL = 'transaction/'

// QUICKBOOKS
export const ASSET_LIABILITIES_AND_EQUITY_URL = 'quickbooks-asset-liabilities-and-equity/'

// FRESHBOOKS
export const PROFIT_LOSS_URL = 'freshbooks-profit-loss/'

// CRS
export const CRS_CONSUMER_CREDIT_REPORT_AND_SCORE = 'crs_consumer_credit_score/'

// 3RD PARTY API ENDPOINTS

// EXPERIAN
export const EXPERIAN_AUTH_URL = 'https://sandbox-us-api.experian.com/oauth2/v1/token'
export const EXPERIAN_SCORE_URL = 'https://sandbox-us-api.experian.com/eits/gdp/v1/request?targeturl=https://sandbox-us-api.experian.com/businessinformation/businesses/v1/scores'
export const EXPERIAN_SCORE_DJANGO_URL = 'http://localhost:8000/experian_app/api/experian_score/'